pkg/patterns/declarative/reconciler.go

```diff
@@ -173,6 +183,43 @@ func (r *Reconciler) reconcileExists(ctx context.Context, name types.NamespacedN
 		return reconcile.Result{}, fmt.Errorf("error applying manifest: %v", err)
 	}
 
+	statusMap := make(map[status.Status]bool)
+	for _, object := range objects.Items {
+
+		unstruct, err := getObjectFromCluster(object, r.config, r.dynamicClient)
+		if err != nil {
+			return reconcile.Result{}, err
+		}
+
+		res, err := status.Compute(unstruct)
+		if err != nil {
+			log.WithValues("unstuctured", unstruct).Error(err, "Unable to get status of unstructured")
+			statusMap[status.NotFoundStatus] = true
```

Comment:
Is NotFoundStatus the best match?  I'm wondering if we have an "internal error" status

Reply:
There is `unknown` status

Reply:
Unknown is probably better - we don't want other systems to draw incorrect conclusions and NotFound feels very transient.  Another option would be to post no status at all for the whole CR, but I feel like unknown is better.